src/core_plugins/kibana/public/dashboard/partials/save_dashboard.html

```diff
@@ -11,12 +11,18 @@
     placeholder="Dashboard title"
     input-focus="select"
   >
+
+  <saved-object-save-as-check-box saved-object="opts.dashboard"></saved-object-save-as-check-box>
+
   <div class="form-group">
     <label>
       <input type="checkbox" ng-model="opts.dashboard.timeRestore" ng-checked="opts.dashboard.timeRestore">
-      Store time with dashboard
+      Store time with {{opts.dashboard.getDisplayName()}}
```

Comment:
Can we change the next line to:

```javascript
<kbn-info placement="bottom" info="Change the time filter to the currently selected time each time this dashboard is loaded"></kbn-info>
```

Currently, this tooltip gets cropped:

![image](https://cloud.githubusercontent.com/assets/1238659/20502647/e1b2c452-aff3-11e6-8477-8d1208ae6014.png)